azavea/tilegarden
# Feature/docker image

## Overview

This PR adds the creation of a docker image from the tiler container which is then published along with the demo site on merges to develop. The `terraform` container has been removed from the development environment and it's functionality has been added to the `tiler` container, so that all deployment steps can be done from a single image. Main functionality for `build-xml`, `publish`, and `destroy` has also been moved into the `tiler` container to free users from requiring extra external scripts.


### Demo

Optional. Screenshots, `curl` examples, etc.


### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.


## Testing Instructions

 * How to test this PR
 * Prefer bulleted description
 * Start after checking out this branch
 * Include any setup required, such as bundling scripts, restarting services, etc.
 * Include test case and expected output


Resolves #XXX
